cafebabel/static/js/articles-tags.js

```diff
@@ -1,42 +1,47 @@
 class Tags {
-  constructor() {
-    this.context = document.querySelector('.tags')
-    this.values = Array.from(this.list.querySelectorAll('input')).map(
-      input => input.value
-    )
-  }
-  get language() {
-    const languages = document.querySelector('#language')
-    return languages.options[languages.selectedIndex].value
-  }
-  get list() {
-    return this.context.querySelector('.tags-list')
+  get context() {
+    return document.querySelector('.tags')
   }
   get removeButtons() {
-    return this.list.querySelectorAll('li button')
+    return this._list.querySelectorAll('li button')
   }
   get fieldAdd() {
     return this.context.querySelector('input[name=tag-new]')
   }
-  get buttonAdd() {
+  get addButton() {
     return this.context.querySelector('button.add')
   }
   get suggestions() {
+    return this._suggestionsContainer.querySelectorAll('li')
+  }
+  get _languages() {
```

Comment:
Not a big fan of the _wrap-everything_ approach. It does abstract the dom in a good way and would make the code more generic, but it duplicates the lines and the code is not generic in the end anyways.

It does make sense for actual verbe methods (`addNewTag`, `removeTag`) though